### ISSUE TITLE
Difference between nilearn MultiPCA and sklearn PCA

### ISSUE THREAD

**Issue by tanweihou** [nilearn/nilearn#1254]
Hi all,

I have a set of 3D PET images of brain and I did PCA on them using both nilearn MultiPCA and sklearn PCA. 

I wanted to duplicate nilearn results using sklearn. For nilearn MultiPCA, I'm just needed to supply the brain mask and the list of images' file path to do the PCA. For sklearn, I have to manually extract the brain images (using the same mask) and then reshape them into 1D array, finally stack them together to do PCA.

However, the results of both methods do not agree with each other. Is there any hidden preprocessing steps nilearn takes in its MultiPCA method ?

Thanks.
